context/app/static/js/components/detailPage/ContributorsTable/utils.ts

```diff
@@ -18,21 +27,29 @@ type V2SchemaId = '94dae6f8-0756-4ab0-a47b-138e446a9501';
 // Add future CEDAR schema IDs here as needed
 type CEDARSchemaID = V2SchemaId;
 
-export interface CEDARContributor {
-  affiliation: string;
+export interface CEDARContributor extends ContributorBase {
   display_name: string;
   email: string;
-  first_name: string;
+  orcid: string;
   is_contact: CEDARBoolean;
   is_operator: CEDARBoolean;
   is_principal_investigator: CEDARBoolean;
-  last_name: string;
   metadata_schema_id: CEDARSchemaID;
-  middle_name_or_initial: string;
-  orcid: string;
 }
 
-export type ContributorAPIResponse = LegacyContributor | CEDARContributor;
+export type ContributorAPIResponse = V0Contributor | LegacyContributor | CEDARContributor;
+
+export interface v0Contact extends V0Contributor {
+  is_contact: true;
+}
+export interface LegacyContact extends LegacyContributor {
+  is_contact: 'TRUE';
+}
+export interface CEDARContact extends CEDARContributor {
+  is_contact: 'Yes';
+}
```

Comment:
I thought there were cases in which `is_contact` wasn't defined for contacts. Was that in `v0`?

Reply:
Yes- just for the V0 case. In that case would the `V0Contact` and `V0Contributor` be identical?

Reply:
Yep!